src/Symfony/Component/Validator/Constraints/ExactlyValidator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+
+/**
+ * @author Hamza Amrouche <hamza.simperfit@gmail.com>
+ */
+class ExactlyValidator extends ConstraintValidator
```

Comment:
I am actually not convinced that we need this validator. What would be a use case for it?

Reply:
because of https://github.com/symfony/symfony/pull/31196#discussion_r291801976